oneanddone/tests/functional/conftest.py

```diff
@@ -71,3 +76,10 @@ def task(base_url, is_local):
     if is_local:
         from oneanddone.tasks.tests import TaskFactory
         return TaskFactory.create()
+
+
+@pytest.fixture(scope='session')
```

Comment:
I'm not familiar enough with Django to help with creating a task that has an assignee. You could do it via the UI but that would add to the test duration and perhaps make it less robust. Looking at `TaskFactory` it seems the owner is set.. perhaps there's a way to set an assignee? https://github.com/mozilla/oneanddone/blob/master/oneanddone/tasks/tests/__init__.py#L65-L81